frontend/src/views/ClientListView.vue

```diff
@@ -4,15 +4,21 @@ import Card from "primevue/card";
 import InputText from "primevue/inputtext";
 import { onMounted } from "vue";
 import { ref } from "vue";
+import { useRouter } from "vue-router";
 
-import { getClients } from "@/backendClient";
+import { deleteClient, getClients } from "@/backendClient";
 import type { Client } from "@/global/types/entities";
 import { ROUTES } from "@/router";
 
+const router = useRouter();
 const clientsList = ref<Client[]>([]);
-
 const value = ref(null);
 
+// we should provide an argument in the place where we call the function
```

Comment:
We already provide the argument, no?

Reply:
just a comment for myself

Reply:
I am not sure I understand. If this is something you would want to fix later I recommend using a prefix like // TODO: ...

But then it looks to me like you already implemented this 🤔